pkg/controller/usersignup/usersignup_controller.go

```diff
@@ -0,0 +1,195 @@
+package usersignup
```

Comment:
could you modify the commit to mark this file here (`pkg/controller/usersignup/usersignup_controller.go`) as a replacement of `pkg/controller/userprovisionrequest/userprovisionrequest_controller.go`? It's much easier to track small changes rather than having to review the whole file again

Reply:
The previous file (userprovisionrequest_controller.go) was essentially a fresh boilerplate controller generated by operator-sdk, there was no custom code in it.